README.md

```diff
@@ -21,3 +21,11 @@ $ python weather_check.py 15 51.123456 -1.123456 --secret SomeSecretKeyHere!
 2017-01-06 09:58:17,943 Starting new HTTPS connection (1): api.darksky.net
 2017-01-06 09:58:18,350 You don't need to worry about salt tonight.
 ```
+
+## Prerequisites:
+
+This script requires Python and the argparse, logging & requests Python modules. For Ubuntu only the requests module is not included with the basic Python pacakges.  It can be installed by:
+
+```
+sudo apt install python python-requests
```

Comment:
> Its usually preferable to use the Python package Index to install python libraries via the pip tool 

That is a matter of some debate on Ubuntu and Debian systems that offer alternative packaging methods.  The basic Python install comes with apt packaged modules already so I stuck with that as there seemed to be a supported package that worked. 

> Note also that Ubuntu and most other linux distributions ship with some version of python by default  

That is true.  It might be a requierment for apt.  I am sure one of the scripting languages was quite dangerous to remove from Debian and its descendants.   It seemed safe enough to add it as there is almost certainly a dependency on the python-requests package anyway.